asv/repo.py

```diff
@@ -1,12 +1,9 @@
-# -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-
 import datetime
 
 from . import util
 
-
```

Comment:
This one is needed. In short, the standard is that classes and functions should have two blank lines to separate them, the rest should be separated maximum by one. This is the full standard: https://www.python.org/dev/peps/pep-0008/